api/services/ChatService.js

```diff
@@ -13,11 +13,7 @@ function getChats() {
 }
 
 function addUserMessage(chat) {
-  if (!chat.username) {
-    chat.username = 'Anonymous';
-  }
-  chat.type = 'user';
-  chat.time = new Date(chat.time).toISOString()
+  chat.time = new Date().toISOString();
```

Comment:
Agreed, but that's more in sync with the machine messages generated by the server. What's the case where the client is better than the server for those times?